package.json

```diff
@@ -32,7 +35,7 @@
     "prettier": "1.19.1",
     "prettier-plugin-solidity": "1.0.0-alpha.49",
     "pretty-quick": "^2.0.1",
-    "solidity-coverage": "^0.7.2",
+    "solidity-coverage": "^0.7.9",
```

Comment:
any specific reason to bump this version?

Reply:
I think I fat fingered installing `solidity-coverage` here...